src/main.js

```diff
@@ -52,14 +52,25 @@ const zipFunction = async function(srcPath, destFolder, { skipGo = true, zipGo =
   if (runtime === 'go') {
     if (zipGo) {
       const destPath = join(destFolder, `${filename}.zip`)
-      await zipGoExe(srcPath, destPath, filename, stat)
+      await zipBinary(srcPath, destPath, filename, stat)
       return { path: destPath, runtime }
     } else {
       const destPath = join(destFolder, filename)
       await cpFile(srcPath, destPath)
       return { path: destPath, runtime }
     }
   }
+
+  if (runtime === 'rs') {
```

Comment:
Would it be possible to add `rs` to the `README.md` documentation of the `runtime` property?